org.civicrm.reqangular/test/services/api/option-group.spec.js

```diff
@@ -0,0 +1,98 @@
+/* eslint-env amd, jasmine */
+
+define([
+  'common/angularMocks',
+  'common/models/option-group',
+  'common/mocks/services/api/option-group-mock'
+], function () {
+  'use strict';
+
+  describe('api.option-group', function () {
+    var $q, methods, OptionGroupAPI;
+
+    beforeEach(function () {
+      module('common.models', 'common.mocks');
+    });
+
+    beforeEach(inject([
+      'api.optionGroup',
+      function (_optionGroupAPI_) {
+        OptionGroupAPI = _optionGroupAPI_;
+      }
+    ]));
+
+    beforeEach(inject(function (_$q_) {
+      $q = _$q_;
+      methods = Object.keys(OptionGroupAPI);
+    }));
+
+    it('has expected interface', function () {
+      expect(methods).toContain('valuesOf');
```

Comment:
Not because of TDD, there were no TDD in this case since the spec was just missing. I just copied this block from somewhere. Now I look at it and it does not make sense since this exists:

```js
beforeEach(function () {
  OptionGroupAPI.valuesOf(optionGroupName);
```

which already checks if it exists and it is a function.